apps/docs/src/app/platform/component-docs/platform-forms/checkbox/platform-checkbox-examples/platform-binary-checkbox.component.html

```diff
@@ -1,16 +1,16 @@
 <h3>Binary Checkbox in Template Driven Form</h3>
 <fdp-form-group [multiLayout]="true" [noLabelLayout]="false">
-    <fdp-form-field [id]="'yellow'" [label]="'Yellow'" [noLabelLayout]="true" zone="zLeft" rank="4" [editable]="true">
+    <fdp-form-field [id]="'yellow'" [label]="'Yellow'" [noLabelLayout]="true" rank="4" [editable]="true">
```

Comment:
please don't use unnecessary `[]` bindings and if you see one please remove them (could be done via regexp replace)
it's not only hurts performance for no good reason, but additionally consumers of our lib should not do it too

Reply:
relative to all modified files, please ;)